src/NavItem.js

```diff
@@ -28,10 +28,10 @@ class NavItem extends React.Component {
   }
 
   handleClick(e) {
-    if (this.props.onSelect) {
+    if (this.props.disabled) {
```

Comment:
This also changes the logic to unconditionally `preventDefault` on `disabled`, even with no `onSelect`?

Reply:
I mean, what's wrong with preventing a user from doing a default action on a disabled element?